src/pretix/presale/templates/pretixpresale/event/fragment_subevent_calendar.html

```diff
@@ -1,41 +1,62 @@
 {% load i18n %}
 {% load eventurl %}
 {% load urlreplace %}
-<form class="form-inline" method="get" id="monthselform" action="{% eventurl event "presale:event.index" cart_namespace=cart_namespace %}">
-    {% for f, v in request.GET.items %}
-        {% if f != "month" and f != "year" %}
-            <input type="hidden" name="{{ f }}" value="{{ v }}">
-        {% endif %}
-    {% endfor %}
     <div class="row">
         <div class="col-sm-4 hidden-xs text-left flip">
-            <a href="?{% url_replace request "year" subevent_list.before.year "month" subevent_list.before.month %}"
-                    class="btn btn-default">
-                <span class="fa fa-arrow-left" aria-hidden="true"></span>
-                {{ subevent_list.before|date:"F Y" }}
-            </a>
+            <form method="get" action="{{ request.path }}">
```

Comment:
Converting this (any the many others like it) from a link into a form will probably stop Google and other search engines from scraping the calendar which might negatively impact future events to show up in search engines. Do we want that? Do we need to make this change?

Reply:
No, not really. I think they are fine as links – even just to see where it is taking me. The only reason I changed it, was because it was mentioned in the audit that these UI-elements need to be <button>s. But I will try to debate whether this is really true.

Reply:
It might be the wording, A11y-tester wanted an „action“ – e.g. „to previous month“. Still waiting for feedback regarding my new wording as an aria-label on the link: „to previous month (April 2021)“.

Reply:
Feedback confirms link, button is just a recommendation and is not a violation any more. I will revert to link.